lib/suspenders/generators/app_generator.rb

```diff
@@ -14,6 +14,9 @@ class AppGenerator < Rails::Generators::AppGenerator
     class_option :heroku_flags, type: :string, default: "",
       desc: "Set extra Heroku flags"
 
+    class_option :ci, type: :string, default: "circle",
+      desc: "Specify a CI provider (options: circle, travis, codeship)"
```

Comment:
Just following the style throughout the file -- happy to revert to what Hound suggests